### DISCUSSION TITLE
Why 'DataValidationErrors' will be triggered first time I open the flyout when I set 'Required' to binding property?

### DISCUSSION THREAD

**Discussion by jim-jiang-github** [AvaloniaUI/Avalonia#11536] [Q&A] [answered]
https://github.com/AvaloniaUI/Avalonia/assets/29189410/6c98c875-0118-47f2-bce1-fda6ab038609

Here is the demo: https://github.com/[jim-jiang-github/LogVisualizer](https://github.com/jim-jiang-github/LogVisualizer)
I want to check DataValidationErrors after Input any thing or click the Comfirm button.

**Comment by maxkatz6** [comment, marked as answer]
It works as expected, as validation works on the property value. Which can be there from any kinds of sources.
You can make "null" as default value, and ignore errors when it's null.
